client/post-editor/post-editor.jsx

```diff
@@ -1086,7 +1086,7 @@ export class PostEditor extends React.Component {
 
 			// `this.findBookmarkedPosition` inserted some markup into the TinyMCE content.
 			// Reset it back to the original.
-			this.editor.setEditorContent( content );
+			//this.editor.setEditorContent( content );
```

Comment:
Unfortunately, one does not simply comment out this line. I does not just remove the bookmark markup. (I authored the comment when I didn't know yet and I'm sorry for that). It also synchronizes the Visual Mode `iframe` content into the HTML Mode `textarea` when switching from Visual to HTML.

Consider the following steps:
1. In HTML mode, paste `<p style="padding-left:30px;">hi</p>`
2. Switch to Visual mode and edit the "hi" into "hi five"
3. Switch back to HTML mode and observe that the content has not been updated -- still just a "hi".

If I uncomment the line, we are back where we were in the beginning: the Visual edit is synced into the HTML mode, but then also the space in the `style` attr is added.